lib/yaml_normalizer/services/normalize.rb

```diff
@@ -54,6 +57,21 @@ def stable?(yaml_a, yaml_b)
         end
       end
 
+      # Psych parser sometimes adds / removes whitespace at the end of the file which causes confising changes in the
```

Comment:
Also, if you see unexpected behaviour in Psych ("sometimes adds / removes whitespace"), please raise an issue [there](https://github.com/ruby/psych/issues) instead of working around it in this project.